frontend/javascripts/oxalis/view/layouting/tracing_layout_view.tsx

```diff
@@ -123,16 +123,27 @@ class TracingLayoutView extends React.PureComponent<PropsWithRouter, State> {
 
   componentWillUnmount() {
     // Replace entire document with loading message
-    // @ts-expect-error ts-migrate(2531) FIXME: Object is possibly 'null'.
-    document.body.removeChild(document.getElementById("main-container"));
+    if (document.body != null) {
+      const mainContainer = document.getElementById("main-container");
+      if (mainContainer) {
+        document.body.removeChild(mainContainer);
+      }
+    }
     window.removeEventListener("resize", this.debouncedOnLayoutChange);
-    const refreshMessage = document.createElement("p");
+
+    const refreshMessageContainer = document.createElement("div");
+    refreshMessageContainer.style.display = "grid";
```

Comment:
@daniel-wer I sneaked this change in this PR, since it caught my eye during testing. just as a heads-up ^^

Reply:
Awesome :tada: